### DISCUSSION TITLE
MQTT camera_name/motion vs camera_name/motion/state

### DISCUSSION THREAD

**Discussion by MacgyverH** [blakeblackshear/frigate#4889] [General]
I noticed there are 2 entries for MQTT for motion. Could someone explain the differences between `camera_name/motion` ON/OFF value vs `camera_name/motion/state` value?

is one if motion detection is on, and the other is motion detected? or is this a bug?

Lines for reference:

https://github.com/blakeblackshear/frigate/blob/ebef38e4088b09f7839b324e228229b9af71b845/frigate/comms/mqtt.py#L56-L60

https://github.com/blakeblackshear/frigate/blob/ebef38e4088b09f7839b324e228229b9af71b845/frigate/comms/mqtt.py#L76-L80

**Comment by NickM-27** [comment]
This is covered in the MQTT docs, one is a switch to enable or disable motion detection and the other is a motion detector.

**Comment by MacgyverH** [comment]
Sigh.. I looked right over that.

Thanks for taking the time to answer!
